pkg/util/log.go

```diff
@@ -9,16 +9,15 @@ import (
 	"github.com/go-kit/kit/log/level"
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/weaveworks/common/logging"
-	"github.com/weaveworks/common/middleware"
 	"github.com/weaveworks/common/server"
 
-	"github.com/cortexproject/cortex/pkg/tenant"
+	"github.com/cortexproject/cortex/pkg/util/logutil"
 )
 
 var (
 	// Logger is a shared go-kit logger.
 	// TODO: Change all components to take a non-global logger via their constructors.
-	Logger = log.NewNopLogger()
+	Logger = logutil.Logger
```

Comment:
If we want to keep this var to avoid lot of code changes, let's at least mark it as deprecated.